templates/centos6/elasticsearch.repo.j2

```diff
@@ -1,6 +1,6 @@
-[elasticsearch-1.5]
-name=Elasticsearch repository for 1.5.x packages
-baseurl=http://packages.elastic.co/elasticsearch/1.5/centos
+[elasticsearch-1.6]
```

Comment:
If we are gonna break things we might as well rename it to `elasticsearch`, that way next versions won't break a thing.